samples/cpp/benchmark_app/benchmark_app.hpp

```diff
@@ -129,6 +129,10 @@ static const char report_type_message[] =
 // @brief message for report_folder option
 static const char report_folder_message[] = "Optional. Path to a folder where statistics report is stored.";
 
+// @brief message for json_stats option
+static const char json_stats_message[] =
+    "Optional. Enables JSON-based statistics output (by default reporting system will use CSV format).";
```

Comment:
Maybe it's better to indicate that the flag must be applied along with `-report_type`.

Reply:
Done